build-system/pr-check.js

```diff
@@ -380,6 +380,12 @@ const command = {
   buildValidator: function() {
     timedExecOrDie('gulp validator');
   },
+  checkBundleSize: function(compiled) {
+    const args = compiled
+      ? '-f "./dist/v0.js" -s "75 kB"'
+      : '-f "./dist/amp.js" -s "315 kB"';
+    timedExecOrDie(`npx bundlesize ${args}`);
```

Comment:
Based on https://github.com/siddharthkp/bundlesize#cli, shouldn't this be...
```
timedExecOrDie(`bundlesize ${args}`);
```
... without the `npx`?

Reply:
That only works if `bundlesize` is installed as a global package. Check out npx: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b